docs/competency-api/audience.md

```diff
@@ -4,4 +4,46 @@ title: Audience
 nav_order: 2
 parent: Competency API
 ---
-# Audience
\ No newline at end of file
+# Audience
+## Update Audience
+
+### Definition
+```HTTP
+https://api.competency-constructor.clark.center/competencies/:competencyId/audience
+```
+
+## Headers
+```JSON
+{
+    Bearer: Bearer.Example.bearer.token
```

Comment:
I think our Bearer uses a space not a . so "Bearer jaspnapsfha.aljnsdpaos.asljdnfpoa" so that we can split(" ") and not split the token